cli/dump.go

```diff
@@ -9,17 +9,42 @@ import (
 	cli "github.com/urfave/cli/v2"
 )
 
+type DumpFormat = string
+
+const (
```

Comment:
Nit: seeing this already belongs to a type called `DumpFormat` and is contained inside `dump.go`, it feels a little redundant to prefix all of the values with `dump` lol.